main-server/src/models/challenge.rs

```diff
@@ -202,9 +205,23 @@ impl HomePageChallenge {
                 category as "category!: ChallengeCategory",
                 scores.score,
                 CAST(description AS varchar(120)) as "description!",
-                post_mortem_date
+                post_mortem_date,
```

Comment:
When editing queries, you need to run:
```
cargo sqlx prepare --workspace
git add .sqlx
```
This will add the type information for the query to the repo